packages/fxa-payments-server/src/routes/Product/PlanDetails/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Plan } from '../../../store/types';
+
+import './index.scss';
+
+export type PlanDetailsProps = { plan: Plan };
+
+// Table of lazy-loaded plan detail components - the keys are plan IDs.
+type availableDetailsType = {
+  [propName: string]: React.LazyExoticComponent<(props: PlanDetailsProps) => JSX.Element>
```

Comment:
I felt compelled to look up ExoticComponent because of this [but didn't feel any the wiser in the end](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/7ee6546bde3962f3bd030202c3ad6fe757a402b6/types/react/index.d.ts#L300-L309)

Reply:
Admittedly, I don't really know what it is either. 😅 I got this by hovering over the lazy-loaded components in App.tsx and copypasta'd the type that came up.